playwright/src/main/java/com/microsoft/playwright/junit/impl/ExtensionUtils.java

```diff
@@ -49,6 +51,13 @@ static boolean isParameterSupported(ParameterContext parameterContext, Extension
   static void setTestIdAttribute(Playwright playwright, Options options) {
     String testIdAttribute = options.testIdAttribute == null ? "data-testid" : options.testIdAttribute;
     playwright.selectors().setTestIdAttribute(testIdAttribute);
+  }
 
+  private static Class<?> getOuterClass(Class<?> clazz) {
+    if (clazz.getDeclaringClass() == null) {
```

Comment:
Should it be `getEnclosingClass` to work for anonymous nested classes too?

Reply:
I don't think you can have anonymous inner classes as test classes. I could be wrong though.

Reply:
I can update it, I don't think it hurts anything

Reply:
> I don't think you can have anonymous inner classes as test classes. I could be wrong though.

Now that I think about it, I believe you are right. The only way to declare an anonymous class would be to create an instance of it, but one won't be able to mark it as @Nested.